tensorboard/webapp/metrics/effects/index.ts

```diff
@@ -195,23 +236,31 @@ export class MetricsEffects implements OnInitEffects {
     fetchInfos: CardFetchInfo[],
     experimentIds: string[]
   ) {
-    /**
-     * TODO(psybuzz): if 2 cards require the same data, we should dedupe instead of
-     * making 2 identical requests.
-     */
-    const requests: TimeSeriesRequest[] = fetchInfos.map((fetchInfo) => {
-      const {plugin, tag, runId, sample} = fetchInfo;
-      const partialRequest: TimeSeriesRequest = isSingleRunPlugin(plugin)
-        ? {plugin, tag, runId: runId!}
-        : {plugin, tag, experimentIds};
-      if (sample !== undefined) {
-        partialRequest.sample = sample;
-      }
-      return partialRequest;
-    });
-
     // Fetch and handle responses.
-    return of(requests).pipe(
+    return this.tagToEid$.pipe(
```

Comment:
Rather than piping this.tagToEid$ can we just get the latest value? I'm a little worried about subtle bugs when tagToEid$ changes for whatever unpredicatable reason and the pipe kicks off a new set of requests.

Reply:
I've added a `take(1)` I could use a subject instead or maybe a subscription? Let me know if you'd prefer something else.